docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | Arbitrary string (acts as a token) | Triggers reconciliation of the resource when the value changes. |
```

Comment:
"Arbitrary string (acts as a token)" probably does not impart adequate guidance on how to use this. Maybe try something like this:

```suggestion
| `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | A string that is unique or at least unlikely to repeat, such as a UUID or a timestamp of "now" | Triggers reconciliation of the resource when its value changes. |
```